lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9780,7 +9780,8 @@ static void loadAllMembersOfRecordDecl(StructDecl *recordDecl) {
       continue;
 
     for (auto found : recordDecl->lookupDirect(name)) {
-      if (addedMembers.insert(found).second)
+      if (addedMembers.insert(found).second &&
```

Comment:
Small thing: If you still have the branch open, do you mind adding a comment saying that this can happen when a member is found inside of an extension? Otherwise, someone like me (or future me) might think "why aren't we correctly filtering these decls in lookupDirect?"

I'm also happy to add the comment in a follow up PR. I have a few cleanups I need to make today anyway.

Reply:
Ahh, I don't have the branch still separate, sorry.